auto-generated/cluster/stone-prd-rh01/managed/rh-managed-cnv-fbc-tenant/appstudio.redhat.com_v1alpha1_releasestrategy_fbc-strategy.yaml

```diff
@@ -1,32 +0,0 @@
-apiVersion: appstudio.redhat.com/v1alpha1
-kind: ReleaseStrategy
-metadata:
-  name: fbc-strategy
-  namespace: rh-managed-cnv-fbc-tenant
-spec:
-  params:
-  - name: fromIndex
-    value: quay.io/scoheb/fbc-index-testing:latest
-  - name: targetIndex
-    value: quay.io/scoheb/fbc-target-index-testing:latest
-  - name: requester
```

Comment:
@scoheb what was this for? It isn't in the new RPA's data, and I don't even see it as a pipeline parameter before the change for the API rework. Same with overwriteFromIndex

Reply:
probably leftover from before your standing auth work. This is being removed since it is was part of a ReleaseStrategy and not added to RPA.

Reply:
Yes they are.